static/js/containers/SignupDialog.js

```diff
@@ -58,12 +58,18 @@ const SignupDialog = ({
         Continue with edX
       </a>
       <div className="terms-of-service-text">
-        By clicking "Continue with edX" I certify that I agree with <a href="/terms_of_service" target="_blank">
+        {'By clicking "Continue with edX" I certify that I agree with '}
```

Comment:
yes - with the tags added the line was too long so I had to rearrange it slightly. the `{'string '}` nonsense is just to deal with the JSX whitespace issues.